### PR TITLE
Fixed: Add section on inventory sidecar files

### PR DESCRIPTION
**Pull request by ahankinson**
This PR adds a section on hash sidecar files.

Refs #29

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,6 +209,21 @@ <h2>Basic Structure</h2>
     <h2>Digest Algorithm Choice</h2>
   </section>
 
+  <section id='digest-sidecar-file'>
+    <h2>Digest Sidecar File</h2>
+    <p>
+        Every occurrence of an inventory file MUST have an accompanying sidecar file stating its checksum. This sidecar file must be of the form <pre>inventory.jsonld.HASHALG</pre>, where <pre>HASHALG</pre> follows the chosen hashing
```
Comment (awoods) [inline index.html]: Is there a canonical list of hash algorithms we can reference?

Comment (zimeon) [reply index.html]: See comments on #21 -- I think the answer is a bit tricky but we will need to define the list, hopefully by reference to one or more other specs. I would suggest we merge this and solve that as a separate problem

**Comment by zimeon** [comment]
Is the addition of the filename following parallel with the bagit approach? I don't object to that but it seems unnecessary

**Comment by awoods** [comment]
@zimeon : https://tools.ietf.org/html/draft-kunze-bagit-08#section-2.1.3

**Comment by zimeon** [comment]
Yes, I see it there and there it is necessary because the `manifest-XYZ.txt` file contains checksums for multiple files. However, we have a different naming convention and the checksum for just one file... so it isn't really necessary

**Comment by ahankinson** [comment]
It is also used in the output of `shasum`:

```
$>shasum -a 512 CNAME
dba3b3255390e9bda6fcc1f3cc8534a4c9fec4f90478c8e62bfaae73fab5700d524a8fb09d692c41cdfd83ec978a282082b586392325124c7cd5c21cca115599  CNAME
```

and is also used in software distributions as sidecars, e.g.,

http://www-eu.apache.org/dist/lucene/solr/7.4.0/

(although I'm not sure what the * in the filename means).

**Comment by zimeon** [comment]
Thanks @ahankinson - I agree the * is odd but good to see similar use in other places

**Comment by ahankinson** [comment]
I updated per #35 so re-review and approval is necessary.

**Comment by ahankinson** [comment]
@zimeon This section in the output of the `shasum` utility explains what the * is for:

> The sums are computed as described in FIPS-180-4.  When checking, the
input should be a former output of this program.  The default mode is to
print a line with checksum, a character indicating type (`*' for binary,
` ' for text, `?' for portable, `^' for BITS), and name for each FILE.

So maybe we also need to decide the mode?

**Comment by zimeon** [comment]
Will port your comment above onto #39

**Pull request merged**